src/utils/storageUtils/index.js

```diff
@@ -0,0 +1,8 @@
+const EXTRA_STORAGE_ADDRESS_MAP = {
+  "0x5f23Cc6F34640396b0B7023292bee7dc4C2b137B": 100000000, // 100MB in bytes
```

Comment:
Can you store and pull this info from database?

Reply:
Done @vijaykrishnavanshi